about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2020-01-25 17:00:00 -0700
+---
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+`<Four different levels of getting agreement/acknowledgement, from easiest to most onerous>`
+
+In order to ~~emulate the automatic EULAs of big, powerful software companies~~ reduce the friction of adding data to the SGID, by completing a [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID, you agree to this Data Sharing Agreement.
+
+—OR—
+
+You accept this Data Sharing Agreement by checking the appropriate checkbox in the [Porter issue](https://github.com/agrc/porter/issues) tracking your SGID data submission.
+
+—OR—
+
+You must visit `<Google Form/other web-based form and response tracker>` and agree to this Data Sharing Agreement prior to creating a new [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID.
+
+—OR—
+
+You must submit a signed copy of this Data Sharing Agreement to our office prior to creating a new [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID.
+
+## Definitions
+{: .text-left}
+
+**"Aggregate data"** refers to any data that cover a specific area of the state and are provided by another organization (such as a city or county) for UGRC to combine with other similar data into statewide datasets.
+
+**"External data"** refers to any data stored or hosted by a person or organization outside of UGRC.
+
+**"Hosted data"** refers to any data stored, maintained, and served by UGRC, whether created by UGRC or provided by another organization. It may include aggregate data.
+
+**"SGID entry"** refers to any of the following:
+
+1. Links in the SGID Index to external data
+1. Links and references in opendata.gis.utah.gov to external data in the form of ArcGIS Online items owned by other organizations
+1. Hosted data in the Open SGID
+1. Hosted data in UGRC's ArcGIS Online organization that are shared with an SGID ArcGIS Online group or the "AGRC Shelf" group
+
+The **"steward"** refers to the individual or organization that shares data with UGRC for inclusion in the SGID.
+
+## Access and Data Quality Provisions
+{: .text-left}
+
+- UGRC will make all SGID entries available to the general public without any form of login or account.
+- UGRC may edit any hosted data to fix any technical errors that prevent the data from being stored in the relevant databases.
```

Comment:
**nitpick**: i think replacing database with something more generic is an improvement. 

```suggestion
- UGRC may edit any hosted data to fix any technical errors that prevent the data from being shared in the SGID.
```